app/renderer/components/preferences/payment/disabledContent.js

```diff
@@ -59,7 +58,7 @@ const styles = StyleSheet.create({
     backgroundColor: globalStyles.color.lightGray,
     borderRadius: globalStyles.radius.borderRadiusUIbox,
     padding: '40px',
-    fontSize: paymentStyles.font.regular,
+    fontSize: globalStyles.payments.fontSize.regular,
```

Comment:
ok I'm against `payments.js` styles anyway so doing what you did is better up to a point that we remove that file.